.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+# Please don't open any new issues here
+
+We moved our issue tracking to Jira. Please use the following page instead:
+
+[ticketsystem.schul-cloud.org/projects/AN](https://ticketsystem.schul-cloud.org/projects/AN/issues)
+
+(Click the red create button at the top of the page)
```

Comment:
Issues can be deactivated for this repo. So we don't need this.

Reply:
+1 Max, I've deactivated Issues for this Repo.